theano/gpuarray/tests/test_opt.py

```diff
@@ -467,32 +469,30 @@ def test_many_arg_elemwise():
 
                 outputs = []
                 for mode in [mode_with_gpu, mode_without_gpu]:
-                    # test the optijmization local_gpu_elemwise_0
-                    f = theano.function(
-                        symb_args, op_to_test(*symb_args),
-                        mode=mode.excluding("local_gpu_elemwise_1"))
+                    # test the optimization local_gpua_elemwise
+                    output = op_to_test(*symb_args)
+                    f = theano.function(symb_args, output)
```

Comment:
Oops, mode=mode should still be passed otherwise we can't force mode_with_gpu and mode_without_gpu.